Firmware/ESP32/flash_esp32.sh

```diff
@@ -136,4 +142,3 @@ esptool.py --before no_reset --baud $BAUDRATE \
                 0x8000  "$SCRIPT_DIR/build/partition_table/partition-table.bin" \
                 0xe000  "$KEYFILE" \
                 0x10000 "$SCRIPT_DIR/build/openhaystack.bin"
-rm "$KEYFILE"
```

Comment:
@fhessel was the keyhole automatically removed before this edit? Or was it not removed if something failed? 
I'd suggest that the keyfile should be removed in the end, even if the script fails

Reply:
As long as the keyfile is a public key, deleting it in case of failure makes sense. If it should become private key material, then I would only delete it automatically if there is another copy of the key material somewhere or if the key material is  obsolete after failed deployment anyway.